plutus-executables/test/certifier/Spec.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
This pragma is likely not needed as the language extension is already switched on by default in  the cabal file.